topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -477,7 +478,7 @@ Galaxy makes this very easy with the `Extract workflow` option. This means any t
 >
 > 6. Click on **Workflow** in the top menu of Galaxy. Here you have a list of all your workflows. Your newly created workflow should be listed at the top:
 >
->    ![`Your workflows` list](../../images/101_26.png)
+>    ![`Your workflows` list](../../images/workflow_cards.png "Workflow are listed in  the center pane of the Galaxy interface as cards.")
```

Comment:
I appreciate that this is a *much* better file name, could you please remove the replaced images so we can be sure that a) they aren't used anywhere else (as the site will fail to build) and b) if they are, we can update them to also use a newer better picture.